pkg/apis/cr/v1alpha1/repositoryserver_types.go

```diff
@@ -81,13 +81,18 @@ type Repository struct {
 	// PasswordSecretRef has the password required to connect to kopia repository
 	PasswordSecretRef corev1.SecretReference `json:"passwordSecretRef"`
 	CacheSizeSettings CacheSizeSettings      `json:"cacheSizeSettings,omitempty"`
+	// LogDirectory is an optional field to specify kopia log directory
+	LogDirectory string `json:"logDirectory,omitempty"`
+	// ConfigDirectory is an optional field to specify kopia config directory
+	ConfigFilePath string `json:"configFilePath,omitempty"`
 }
 
 // CacheSizeSettings are the metadata/content cache size details
 // that can be used while establishing connection to the kopia repository
 type CacheSizeSettings struct {
```

Comment:
```suggestion
type CacheSettings struct {
```
Should we rename it now since it's not just `CacheSizeSettings` anymore?
Although that could break the CRD so maybe we can add another object, say `Configuration`, under which we can keep adding new fields. WDYT?